src/hooks/events/useGetEvent.ts

```diff
@@ -23,7 +23,8 @@ export function useGetEvent() {
                 filter: filter,
                 orgUnit: orgUnit,
                 trackedEntity: trackedEntity
-            })) as unknown as EventQueryResults;
+            }))
+            .catch((error) => error ) as unknown as EventQueryResults;
```

Comment:
Why you didnt put show alert component

Reply:
I'm doing it on the hook where i call this function

Reply:
Okay, if you're doing on that way, you dont need to put catch function two times, just put one time